ios/RNTwilioVoice/RNTwilioVoice.m

```diff
@@ -555,7 +734,6 @@ - (void)performEndCallActionWithUUID:(NSUUID *)uuid {
   if (uuid == nil) {
     return;
   }
-
   UIDevice* device = [UIDevice currentDevice];
   device.proximityMonitoringEnabled = NO;
```

Comment:
I had to move this to `didDisconnectWithError` or `callDisconnected` for this to work. I'm not sure this is the correct place for the call.